README.md

```diff
@@ -9,3 +9,16 @@ This package is installed using Composer.
 ## Setup
 
 Run `composer require eleven-miles/iris` to install.
+
+### Debugging
+
+For debugging: `Debug::debug($image_meta);`
+
+## Action Scheduler
+
+Iris uses [Action Scheduler](https://actionscheduler.org/usage/) to run webp convertion at scheduled intervals. This functionality helps when converting media libraies, and when images are uploaded in bulk. The GD image processing can become hit memory issues when bulk uploading images and return failed conversion. Action Scheduler helps to ensure that all images are converted by queuing up the conversions.
+
+To update Action Scheduler run the following commands:
+
+`git fetch subtree-action-scheduler trunk`
```

Comment:
Appreciate this might just be best way to update during development, but could this also be achieved via composer package updates?